kubeflow/core/jupyterhub.libsonnet

```diff
@@ -4,16 +4,27 @@
   new(_env, _params):: {
     local params = _env + _params,
 
+    local defaultSpawnerData() = {
```

Comment:
no need to define a function. Just `defaultSpawnerData = ` would work

Reply:
Kam, we have pushed the change you suggested in our PR. Please take a look.